CHANGELOG.md

```diff
@@ -14,6 +14,9 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ## [0.14.29] - 2020-12-21
 - Fixed media items not working
 
+## [0.14.29] - 2020-12-21
```

Comment:
Please update changelog as well.

Reply:
I already updated it.

Reply:
https://github.com/ChildMindInstitute/mindlogger-app/pull/1395/commits/2ba1ddb8b55f2ef2c8a165669d7f21fe31f6022f

Reply:
Your PR was being delayed until it's reviewed and other PRs are already merged with later versions. You need to update the changelog description with the right version and put it in right place.

Reply:
hm, I'll update it.
should we update version and changelog several times like now to merge one PR?

Reply:
It's third time I update the version/changelog ...